tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -846,7 +847,9 @@ func isWebAction(client *whisk.Client, qname QualifiedName) bool {
     if err != nil {
         whisk.Debug(whisk.DbgError, "client.Actions.Get(%s) error: %s\n", fullActionName, err)
         whisk.Debug(whisk.DbgError, "Unable to obtain action '%s' for web action validation\n", fullActionName)
+        errMsg = wski18n.T("API action does not exist")
     } else {
+        errMsg = wski18n.T("API action is not a web-action")
         weVal := getValue(action.Annotations, "web-export")
         if (reflect.TypeOf(weVal) == nil) {
```

Comment:
Would rather not use an unneeded package here in reflect as you can do the nil comparison without it.